changelogs/fragments/unittest-fix-for-resource-module-base.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
Since this is only a test change and doesn't affect end user experience, can we make this a `trivial` changelog?